panic

Running the latest release 0.59.0 and master as of right now.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x13c793f]

goroutine 936 [running]:
github.com/roboll/helmfile/state.findChartDirectory.func1(0xc0004e2000, 0x66, 0x0, 0x0, 0x156b7e0, 0xc00032c8d0, 0x10, 0x146f960)
	/Users/sstarcher/go/src/github.com/roboll/helmfile/state/state.go:1023 +0x2f
path/filepath.Walk(0xc0004e2000, 0x66, 0xc000328320, 0x0, 0xc000328310)
	/usr/local/Cellar/go/1.12.4/libexec/src/path/filepath/path.go:402 +0x6a
github.com/roboll/helmfile/state.findChartDirectory(0xc0004e2000, 0x66, 0x0, 0x0, 0x1, 0xc000328310)
	/Users/sstarcher/go/src/github.com/roboll/helmfile/state/state.go:1022 +0x8d
github.com/roboll/helmfile/state.(*HelmState).downloadCharts.func2(0x1a)
	/Users/sstarcher/go/src/github.com/roboll/helmfile/state/state.go:459 +0x4cc
github.com/roboll/helmfile/state.(*HelmState).scatterGather.func1(0xc000240000, 0xc0000b6180, 0xc0001e4ff0, 0xc0000b61b0, 0x1a)
	/Users/sstarcher/go/src/github.com/roboll/helmfile/state/state_run.go:31 +0x125
created by github.com/roboll/helmfile/state.(*HelmState).scatterGather
	/Users/sstarcher/go/src/github.com/roboll/helmfile/state/state_run.go:29 +0xf1

--- a/state/state.go
+++ b/state/state.go
@@ -1021,7 +1021,10 @@ func chartNameWithoutRepository(chart string) string {
 // find "Chart.yaml"
 func findChartDirectory(topLevelDir string) (string, error) {
 	var files []string
-	filepath.Walk(topLevelDir, func(path string, f os.FileInfo, _ error) error {
+	filepath.Walk(topLevelDir, func(path string, f os.FileInfo, err error) error {
+		if err != nil {
+			return fmt.Errorf("error walking through %s: %v", path, err)
+		}
 		if !f.IsDir() {
 			r, err := regexp.MatchString("Chart.yaml", f.Name())
 			if err == nil && r {

